web/themes/custom/votegov/src/sass/uswds-overrides/usa-banner.scss

```
@@ -15,16 +15,11 @@
 }
```

added this as well to match the design to the `.usa-banner`


```suggestion
  background-color: #F7F9FA;
```